### PR TITLE
Added example of using Kafka Test Containers

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

File: docs/usage/kafka_containers.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Kafka Containers
+
+Test Containers can be used to automatically instantiate and manage [Kafka](https://kafka.apache.org) containers.
```
Comment (bsideup) [inline docs/usage/kafka_containers.md]: `Test Containers` -> `Testcontainers` ;)

Comment (bsideup) [reply docs/usage/kafka_containers.md]: Also, please use `Apache Kafka`, afaik that's how we should refer to it

File: docs/usage/kafka_containers.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+More precisely Test Containers use official Docker images for [Confluent OSS Platform](https://hub.docker.com/r/confluentinc/cp-kafka/)
```
Comment (bsideup) [inline docs/usage/kafka_containers.md]: uses?

File: docs/usage/kafka_containers.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Benefits
+
+* Running single node Kafka installation with just one line of code
+* No need to manage external Zookeeper installation, required by Kafka. But see [below](#zookeeper)
+
+## Example
+
+The following field in your JUnit UI test class will prepare a container running Kafka:
```
Comment (bsideup) [inline docs/usage/kafka_containers.md]: UI test?

Comment (iNikem) [reply docs/usage/kafka_containers.md]: Hrr, never push anything on Friday evening...

File: docs/usage/kafka_containers.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+@Rule
+public KafkaContainer kafka = new KafkaContainer();
+```
+        
+Now your tests or any other process running on your machine can get access to running Kafka broker by using the following bootstrap server location:
+```java
+kafka.getBootstrapServers()
+```
+
+If your test needs to run some other docker container which needs access to the Kafka, do the following:
```
Comment (bsideup) [inline docs/usage/kafka_containers.md]: Could you please move this section to the end of this document? So that only interested parties will read it :D

**Comment by bsideup** [comment]
@iNikem amazing, thank you! 😍 One of the best types of contributions 👍 

I'll keep it open to let @kiview and @rnorth review the wording but I'm super happy with the content, good job 👍

File: docs/usage.md [hunk 5]
```diff
@@ -25,6 +25,7 @@ Testcontainers will try to connect to a Docker daemon using the following strate
 
 * [Temporary database containers](usage/database_containers.md) - specialized Microsoft SQL Server, MariaDB, MySQL, PostgreSQL, Oracle XE and Virtuoso container support
 * [Webdriver containers](usage/webdriver_containers.md) - run a Dockerized Chrome or Firefox browser ready for Selenium/Webdriver operations - complete with automatic video recording
+* [Kafka containers](usage/kafka_containers.md) - run a Dockerized Kafka, a distributed streaming platform
```
Comment (kiview) [inline docs/usage.md]: Using the word `dockerized` as a verb is kind of a [generic trademark](https://en.wikipedia.org/wiki/Generic_trademark), so I think we should write it with a lowercase d. Can you please fix it in the line above as well? :slightly_smiling_face:

File: docs/usage/kafka_containers.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+# Kafka Containers
+
+Testcontainers can be used to automatically instantiate and manage [Apache Kafka](https://kafka.apache.org) containers.
+More precisely Testcontainers uses official Docker images for [Confluent OSS Platform](https://hub.docker.com/r/confluentinc/cp-kafka/)
+
+## Benefits
+
+* Running single node Kafka installation with just one line of code
```
Comment (kiview) [inline docs/usage/kafka_containers.md]: Running a

File: docs/usage/kafka_containers.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+# Kafka Containers
+
+Testcontainers can be used to automatically instantiate and manage [Apache Kafka](https://kafka.apache.org) containers.
+More precisely Testcontainers uses official Docker images for [Confluent OSS Platform](https://hub.docker.com/r/confluentinc/cp-kafka/)
```
Comment (kiview) [inline docs/usage/kafka_containers.md]: uses the

File: docs/usage/kafka_containers.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+Now your tests or any other process running on your machine can get access to running Kafka broker by using the following bootstrap server location:
+```java
+kafka.getBootstrapServers()
+```
+
+## Options
+
+### Selecting Kafka version
+
+You can select a version of Confluent Platform by passing it to container's constructor:
```
Comment (kiview) [inline docs/usage/kafka_containers.md]: to the container's

File: docs/usage/kafka_containers.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+```java
+new KafkaContainer("4.1.2")
+```
+The correspondence between Confluent Platform versions and Kafka versions can be seen [in Confluent documentation](https://docs.confluent.io/current/installation/versions-interoperability.html#cp-and-apache-kafka-compatibility)
+        
+### <a name="zookeeper"></a> Using external Zookeeper
+
+If for some reason you want to use an externally running Zookeeper, then just pass its location during construction:
```
Comment (kiview) [inline docs/usage/kafka_containers.md]: just provide it's location

File: docs/usage/kafka_containers.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+```java
+new KafkaContainer().withExternalZookeeper("localhost:2181")
+```
+
+## Multi-container usage
+
+If your test needs to run some other docker container which needs access to the Kafka, do the following:
```
Comment (kiview) [inline docs/usage/kafka_containers.md]: Docker with uppercase D

File: docs/usage/kafka_containers.md (hunk 9 above)
Comment (iNikem) [reply docs/usage/kafka_containers.md]: I kindly disagree on this one :) https://data.grammarbook.com/blog/pronouns/1-grammar-error/

Comment (kiview) [reply docs/usage/kafka_containers.md]: You are 100% correct of course :D

**Comment by rnorth** [comment]
We have this out in a Release Candidate build (1.9.0-rc1) for anyone who is keen to try it!

[Release notes](https://github.com/testcontainers/testcontainers-java/releases/tag/1.9.0-rc1)

**Pull request merged**